justfile

```diff
@@ -0,0 +1,29 @@
+set quiet
+
+import? '../sdk-codegen/justfile'
+
+_default:
+    just --list
+
+test no_build="" config="Debug":
+    dotnet test {{no_build}} -f net8.0 src/StripeTests/StripeTests.csproj -c {{config}}
+
+# skip build and test in release mode
+test-release: (test "--no-build" "Release")
```

Comment:
This could be simplified if testing just always ran the build command, but I get the sense that in CI we want to build once and use that build repeatedly (for tests, packing, uploading, etc). 

We could also add a `just build` command that was used as a pre-req for packing and uploading, but I'm not too worried about it.